### PR TITLE
Front 38

### PR DESCRIPTION
**Pull request by cecerodrgz**
I don't have the authentication for the login on my branch, but I wanted to push it so everyone could give it a look.
Still working on it but less work is required.

### CODE REVIEW

**Review by Ekaterina-Bondareva — CHANGES_REQUESTED** [review]
1. Could you please add on Profile Page `all user details (address, firstname, lastname, email).`
2. Could you please use the PaginatedResults component as it was pointed in ticket?

**Review by SimonjShurety — APPROVED** [review]
Approved.

**Review by Ekaterina-Bondareva — CHANGES_REQUESTED** [review]
ProfilePage always shows data for currently logged in user even if I use /profile/<userId> URL. This component should work differently depending on value of myProfile property.

Also looks like list of books is duplicated because of <BookList> and <PagePagination> tags on lines 36 and 37 in App.js

File: src/App.js [hunk 1]
```diff
@@ -149,8 +157,11 @@ const App = () => {
                         path="/pagination"
                         element={<PagePagination books={bigBookArray} />}
                     />
+                        <Route path='/my-profile' element={<ProfilePage myProfile={!!sessionObject}/>}/>
+                        <Route path='/profile/:userid' element={<ProfilePage  />} />
```
Comment (Ekaterina-Bondareva) [inline src/App.js:161]: Route to /profile/:userId should be removed

File: src/App.js [hunk 2]
```diff
@@ -149,8 +157,11 @@ const App = () => {
                         path="/pagination"
                         element={<PagePagination books={bigBookArray} />}
                     />
+                        <Route path='/my-profile' element={<ProfilePage myProfile={!!sessionObject}/>}/>
```
Comment (Ekaterina-Bondareva) [inline src/App.js:160]: Could you please remove myProfile property as now ProfilePage will always show only my profile?

File: src/components/ProfilePage/ProfilePage.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
+import { useState, useEffect, useContext } from "react";
+import BookList from "../BookList/BookList";
+import './Profile.scss'
+import { getAllBooksOwnerAdapter } from "../../adapters/book-adapters";
+import { SessionContext } from "../../App";
+import { PagePagination } from "../PagePagination/Pagination";
+
+
+
+const ProfilePage = ({myProfile = false}) => {
```
Comment (Ekaterina-Bondareva) [inline src/components/ProfilePage/ProfilePage.js:10]: Could you please remove `myProfile` property as it is not used?

File: src/components/ProfilePage/ProfilePage.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+       setUsersBooks(booksData.books)
+    }
+
+
+    useEffect( () => {
+        fetchUserBooks()
+    }, [])
+
+    const {sessionObject} = useContext(SessionContext)
+    console.log(sessionObject)
```
Comment (Ekaterina-Bondareva) [inline src/components/ProfilePage/ProfilePage.js:24]: Could you please remove this log statement before merging PR?

**Review by Ekaterina-Bondareva — APPROVED** [review]

**Pull request merged**